src/index.html

```diff
@@ -4,9 +4,381 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo-link">
+        <img
+          src="images/logo.png"
+          alt="Logo image"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a class="nav__list-link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a class="nav__list-link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a class="nav__list-link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#laptops"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a class="nav__list-link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a class="nav__list-link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a class="nav__list-link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a class="nav__list-link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="main__catalog catalog">
+        <div class="catalog__container">
```

Comment:
I think this div is unnecessary because you already have the section

Reply:
Why do you need all this blocks?